README.md

```diff
@@ -10,15 +10,21 @@ Bundler makes sure Ruby applications run the same code on every machine.
 
 It does this by managing the gems that the application depends on. Given a list of gems, it can automatically download and install those gems, as well as any other gems needed by the gems that are listed. Before installing gems, it checks the versions of every gem to make sure that they are compatible, and can all be loaded at the same time. After the gems have been installed, Bundler can help you update some or all of them when new versions become available. Finally, it records the exact versions that have been installed, so that others can install the exact same gems.
 
+To see what has changed in recent versions of Bundler, see the [CHANGELOG](CHANGELOG.md).
+
 ### Installation and usage
 
-To install (or update to the latest version):
+[Visit the Bundler site](http://bundler.io) to access the complete documentation.
```

Comment:
I think this line of instructions should be "above the fold", since its not specific to installation & usage